examples/basic_pipeline.py

```diff
@@ -170,8 +145,10 @@ def basic_pipeline_run(pipeline_class=img.pipelines.MultinestPipeline,
     # Prepares the random magnetic field factory
     B_factory = testFields.NaiveGaussianMagneticFieldFactory(grid=one_d_grid)
     B_factory.active_parameters = ('a0','b0')
-    B_factory.priors ={'a0': img.priors.FlatPrior(interval=[-5, 5]*u.microgauss),
-                      'b0': img.priors.FlatPrior(interval=[2, 10]*u.microgauss)}
+    B_factory.priors ={'a0': img.priors.FlatPrior(xmin=-5*u.microgauss,
+                                                  xmax=5*u.microgauss),
```

Comment:
Why the `xmin` and `xmax` here?
Don't explicitly set them, as it is incredibly obvious for the user that that is what the two arguments are that the class takes.

Reply:
I'll amend this.

Reply:
Actually, I will leave this for #60